src/status/libstatus/stickers.nim

```diff
@@ -137,6 +137,7 @@ proc getPackData*(id: Stuint[256], running: var Atomic[bool]): StickerPack =
     sticker.packId = truncate(id, int))
   result.id = truncate(id, int)
   result.price = packData.price
+  client.close()
```

Comment:
Should this be in a finally to? Even if there is no try/except I would assume this part can still fail (http request) and the connection might stay open